runtime/near-vm-runner/src/imports.rs

```diff
@@ -52,7 +52,7 @@ wrapped_imports! {
     signer_account_pk<[register_id: u64] -> []>,
     predecessor_account_id<[register_id: u64] -> []>,
     input<[register_id: u64] -> []>,
-    block_index<[] -> [u64]>,
+    height<[] -> [u64]>,
```

Comment:
This PR cannot be submitted on its own because it will break a bunch of stuff. It should be accompanied by PRs to nearlib (maybe nearshell), explorer, near-bindgen, near-runtime-ts, and it should include a procedure of hard fork: dumping the state and restarting the nodes+applayer as we discussed in slack. CC @vgrichina @bowenwang1996

Reply:
Sure. Can we work on top of this PR?

Reply:
It depends on what do you mean by working on top of this PR. We can only merge code into staging if it becomes a part of the new release. This code cannot be merged into staging without merging corresponding fixes to other repositories.

Reply:
Yes, the question is to merge into staging or creating a new branch to continue development on top of this commit. I accept any of these decisions. We shouldn't create a huge diff between this PR and actual staging by delaying if possible.

Reply:
Everyone else will continue developing on the top of staging which is also a branch that we use for the releases (approx once a week we merge staging into master and cut a release). We cannot ask everyone to continue working on top of branch other than staging.

I think there is only one way to merge this PR is prepare PRs in other repos and simultaneously do the following:
* merge all PRs;
* dump current network state and restart it with new version -- hard fork.

Reply:
> We cannot ask everyone to continue working on top of branch other than staging.

Why not?

Reply:
Because then it will become a staging branch but with an extra step of inconveniencing everyone by asking them to switch to a different branch from staging.

Merging this PR to staging is equivalent to not merging it and asking everyone to work off this branch.